test/integration/logins-api-test.js

```diff
@@ -46,6 +46,20 @@ describe("logins API", () => {
   let By;
   let until;
 
+  const enableRememberSignons = async () => {
+    await webext.inChrome();
+    await driver.executeScript(`
+      Services.prefs.setBoolPref("signon.rememberSignons", true);
```

Comment:
It would be good to actually call the Services.logins.get/setLoginSavingEnabled for a specific origin, then test it's correctly returned when you call the API. I wonder what happens if you accidentally leave in a trailing slash, for instance. This could be left for a followup commit.

Reply:
From my own experiences, the CRUD portion of the upstream API works just fine regardless of what the global pref and this specific callout are set to.